synth_node_bin/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "synth_node_bin"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1.0"
+clap = { version = "4.1", features = ["derive"] }
+rand = "0.8"
+tokio = "1.25"
+tracing-subscriber = "0.2.0"
```

Comment:
Is it that crate: https://crates.io/crates/tracing-subscriber ?
It's version is v0.3.16.

Reply:
I'm looking sometimes for packages to check what they do :) 
But we can add to readme:
`[![dependency status](https://deps.rs/repo/github/runziggurat/xrpl/status.svg)](https://deps.rs/repo/github/runziggurat/xrpl)`
Just to have status everytime we open the page 😃

Reply:
I'll specify without the `patch` part, it should always be backwards compatible that way

Done

Reply:
> I'm looking sometimes for packages to check what they do :)
But we can add to readme:
[![dependency status](https://deps.rs/repo/github/runziggurat/xrpl/status.svg)](https://deps.rs/repo/github/runziggurat/xrpl)
Just to have status everytime we open the page smiley

yes, please! :D
will do that in the next PR